packages/plugin-dataloader/tests/example/builder.ts

```diff
@@ -11,7 +11,7 @@ export default new SchemaBuilder<{ Context: ContextType }>({
     nodeQueryOptions: {},
     nodesQueryOptions: {},
   },
-  plugins: [RelayPlugin, ErrorsPlugin, DataloaderPlugin],
+  plugins: [ErrorsPlugin, DataloaderPlugin, RelayPlugin],
```

Comment:
Not sure if we should enforce that `RelayPlugin` come before `DataLoaderPlugin` if we detect both are included, otherwise the IDs won't be parsed before loading.

Reply:
I think at some point having some sort of mechanism for ensuring plugin order would be good, but I don't want to address it in the PR.  Ideally the solution would be something generic other plugins can take advantage of.